juicy-checkbox-group-item.html

```diff
@@ -161,7 +161,7 @@
             </template>
             <template is="dom-if" if="{{!checkboxHidden}}">
                 <div id="checkboxItemContainer">
-                    <div id="checkboxItem" data-checkable$="{{checkable}}" data-checked$="{{checked}}" on-click="toggleCheckbox">
+                    <div id="checkboxItem" data-checkable$="{{checkable}}" data-checked$="{{checked}}" on-click="toggleCheckbox" on-keypress="keypressCheckbox" tabindex="0">
```

Comment:
Ah, I did not expect that.

Anyway, `keypress` is a strange event. See: https://www.kirupa.com/html5/keyboard_events_in_javascript.htm

Please check if you could use `keydown` or `keyup`, that would be preferable.